Dockerfile

```diff
@@ -5,9 +5,6 @@ ARG COVERAGE
 RUN if [ "$COVERAGE" = "pcov" ]; then pecl install pcov && docker-php-ext-enable pcov; fi
 
 # Install composer to manage PHP dependencies
-RUN apt-get update && apt-get install -y git zip
-RUN curl https://getcomposer.org/download/1.9.0/composer.phar -o /usr/local/sbin/composer
-RUN chmod +x /usr/local/sbin/composer
-RUN composer self-update
+COPY --from=composer /usr/bin/composer /usr/bin/composer
```

Comment:
Can you raise this in a separate PR with an explanation of the pros/cons please

Reply:
Can do.
I have reverted this commit from the PR.